src/mpyl/reporting/targets/github.py

```diff
@@ -49,12 +50,18 @@
 from ...utilities.repo import Repository, RepoConfig
 
 
+def compose_message_body(results: RunResult, _unused_config: Optional[Dict] = None) -> str:
```

Comment:
What's the point of this `_unused_config`, some sort of python trick that I don't know about?

Reply:
The underscore is a convention for unused parameters that keeps the linter happy.

Reply:
That I know, but why would you pass the unused param in this case?

Reply:
because the function is passed as a parameter. Let me check if you can specify the type of the function 🤔